docs/execution-providers/EP-Context-Design.md

```diff
@@ -80,3 +82,15 @@ OnnxRuntime EPs which support loading from Onnx model with EPContext nodes shoul
   c. If the user loads the model from memory buffer, user needs to provide session option ep.context_file_path. EP gets the folder path from ep.context_file_path, and combines it with the relative path   got from step a) as the context binary file full path. 
 
 <p align="center"><img width="60%" src="../../images/EP_context_nodes_with_different_eps.png" alt="EP Context nodes with different EPs"/></p>
+
+## New ExecutionProvider interface GetEpContextNodes() to help generate the EP Context cache model
+
+It is hard for Execution Providers to generate the partitioned graph whithin the Execution Provider code since Execution Provider does not has a good picture of the whole partitioned graph. New ExecutionProvider interface GetEpContextNodes() is added to support this.
```

Comment:
```suggestion
It is hard for Execution Providers to generate the partitioned graph within the Execution Provider code since an Execution Provider does not have a good picture of the whole partitioned graph. New ExecutionProvider interface GetEpContextNodes() is added to support this.
```